2011/debuggers_part2_code/bp_manual.c

```diff
@@ -23,7 +23,7 @@
 extern char* strsignal(int);
 
 
-void run_debugger(pid_t child_pid)
+void run_debugger(pid_t child_pid, long addr)
```

Comment:
Here and elsewhere, I feel uncomfortable with `long` for addresses. How about `uint64_t`, which is standard in C99?

Reply:
SG